### ISSUE TITLE
Possible random just-made signature expiration

### ISSUE THREAD

**Issue by ni4** [riboseinc/rnp#706]
## Description

Got the following output during local Travis run:
```
/tmp/rnp/build/src/rnp/rnp --homedir /tmp/rnp/build/src/tests/.rnp --pass-fd 4 --userid signing@rnp --sign /tmp/rnp/build/src/tests/cleartext.txt --detach


/tmp/rnp/build/src/rnp/rnp --homedir /tmp/rnp/build/src/tests/.rnp --verify /tmp/rnp/build/src/tests/cleartext.txt.sig
EXPIRED signature made Tue Jul  3 09:27:30 2018
using RSA (Encrypt or Sign) key e91742a71cf0270c
signature  2048/RSA (Encrypt or Sign) e91742a71cf0270c 2018-07-03 [SC] 
                 371a 7877 6dd6 f9f0 ff17 6997 e917 42a7 1cf0 270c 
uid              signing@rnp
Signature verification failure: 1 invalid signature(s), 0 unknown signature(s)
[rnp_process_file() /tmp/rnp/src/lib/rnp.cpp:1323] error 0x12000002

rnp detached verification failed
EXPIRED signature made Tue Jul  3 09:27:30 2018
using RSA (Encrypt or Sign) key e91742a71cf0270c
signature  2048/RSA (Encrypt or Sign) e91742a71cf0270c 2018-07-03 [SC] 
                 371a 7877 6dd6 f9f0 ff17 6997 e917 42a7 1cf0 270c 
uid              signing@rnp
Signature verification failure: 1 invalid signature(s), 0 unknown signature(s)
[rnp_process_file() /tmp/rnp/src/lib/rnp.cpp:1323] error 0x12000002
rnp detached verification failed
EXPIRED signature made Tue Jul  3 09:27:30 2018
using RSA (Encrypt or Sign) key e91742a71cf0270c
signature  2048/RSA (Encrypt or Sign) e91742a71cf0270c 2018-07-03 [SC] 
                 371a 7877 6dd6 f9f0 ff17 6997 e917 42a7 1cf0 270c 
uid              signing@rnp
Signature verification failure: 1 invalid signature(s), 0 unknown signature(s)
[rnp_process_file() /tmp/rnp/src/lib/rnp.cpp:1323] error 0x12000002
ERROR
/home/travis/local-builds/gpg-install/bin/gpgconf --homedir /tmp/rnp/build/src/tests/.gpg --kill gpg-agent
```

Previous iteration was successfull:
```
/tmp/rnp/build/src/rnp/rnp --homedir /tmp/rnp/build/src/tests/.rnp --pass-fd 4 --userid signing@rnp --sign /tmp/rnp/build/src/tests/cleartext.txt --armor --detach


/tmp/rnp/build/src/rnp/rnp --homedir /tmp/rnp/build/src/tests/.rnp --verify /tmp/rnp/build/src/tests/cleartext.txt.asc
Good signature made Tue Jul  3 09:27:29 2018
using RSA (Encrypt or Sign) key e91742a71cf0270c
signature  2048/RSA (Encrypt or Sign) e91742a71cf0270c 2018-07-03 [SC] 
                 371a 7877 6dd6 f9f0 ff17 6997 e917 42a7 1cf0 270c 
uid              signing@rnp
Signature(s) verified successfully

/home/travis/local-builds/gpg-install/bin/gpg --homedir /tmp/rnp/build/src/tests/.gpg --batch --yes --trust-model always --verify /tmp/rnp/build/src/tests/cleartext.txt.asc /tmp/rnp/build/src/tests/cleartext.txt
gpg: Signature made Tue 03 Jul 2018 09:27:29 AM UTC
gpg:                using RSA key 371A78776DD6F9F0FF176997E91742A71CF0270C
gpg: Good signature from "signing@rnp" [unknown]
gpg: WARNING: Using untrusted key!
```
So it could be some problem related to the time granularity.

**Comment by ni4** [comment]
Should be fixed at #723 (just adding the reference).

**Comment by ni4** [comment]
Doesn't seem to be exposed anymore so considering it fixed in #723
